packages/editor/src/components/post-format/index.js

```diff
@@ -38,6 +38,20 @@ export const POST_FORMATS = [
 	return 0;
 } );
 
+/**
+ * `PostFormat` is a React component that provides a dropdown selector for post formats.
```

Comment:
I'd rephrase this a bit to exclude the implementation details (dropdown, fetches etc..) to something like:

... a component that allows changing the post format while also providing a suggestion for the current post.